README.rst

```diff
@@ -11,15 +15,24 @@ PyMODM
    :alt: Join the chat at https://gitter.im/mongodb/pymodm
    :target: https://gitter.im/mongodb/pymodm?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge
 
-A generic ODM around PyMongo_, the MongoDB Python driver. Its goal is to provide
-an easy, object-oriented interface to MongoDB documents. PyMODM works on Python
+A generic ODM around PyMongo_, the MongoDB Python driver. PyMODM works on Python
 2.7 as well as Python 3.3 and up. To learn more, you can browse the `official
 documentation`_ or take a look at some `examples`_.
 
 .. _PyMongo: https://pypi.python.org/pypi/pymongo
 .. _official documentation: https://pymodm.readthedocs.io
```

Comment:
Should this link be changed to `http://pymodm.readthedocs.io/en/stable/?badge=stable`?